local-volume/provisioner/pkg/deleter/jobcontroller.go

```diff
@@ -46,8 +46,8 @@ const (
 	PVLabel = "pv"
 	// PVUuidLabel  is the label name whose value is the pv uuid.
 	PVUuidLabel = "pvuuid"
-	// DeviceAnnotation is the annotation that specifies the device path.
-	DeviceAnnotation = "device"
```

Comment:
Changing this could break any monitoring/logging built on top of this

Reply:
Reverted.
I updated it because the name is a bit confusing now.

Reply:
Yeah it is a little confusing now. Would it make sense to have a new annotation for the filesystem case?